docs/source/conf.py

```diff
@@ -451,6 +451,19 @@
     "seed_all",
     "set_rng_state",
     "set_rng_state_all",
+    # torch.cuda.tunable
```

Comment:
New entries in this file are not allowed anymore.
Please make sure that your public APIs are properly documented (docstring) and visible on the website (.rst properly updated)

Reply:
Any quick pointers on which *.rst file(s) to edit?

Reply:
For cuda APIs like this, I guess a new section in cuda.rst https://github.com/pytorch/pytorch/blob/main/docs/source/cuda.rst

Reply:
Yep yep, I'm on it!

Reply:
Done.  Please see latest commit [529addd](https://github.com/pytorch/pytorch/pull/124362/commits/529adddc71d422656a51db4082e63dfaa290a663).